pkg/subctl/cmd/validate_deployment.go

```diff
@@ -17,46 +17,128 @@ package cmd
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/spf13/cobra"
-	"github.com/submariner-io/submariner-operator/pkg/internal/cli"
+	submarinerv1 "github.com/submariner-io/submariner/pkg/apis/submariner.io/v1"
+	smClientset "github.com/submariner-io/submariner/pkg/client/clientset/versioned"
+	"github.com/submariner-io/submariner/pkg/util"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/rest"
+
+	"github.com/submariner-io/submariner-operator/apis/submariner/v1alpha1"
+	"github.com/submariner-io/submariner-operator/pkg/internal/cli"
 )
 
 var validatePodsCmd = &cobra.Command{
-	Use:   "pods",
-	Short: "Validate the submariner pods",
-	Long:  "This command checks that all the submariner pods are running",
-	Run:   validatePods,
+	Use:   "deployment",
+	Short: "Validate the submariner deployment",
+	Long:  "This command checks if there are any issues related to overlapping CIDRs and the state of submariner pods.",
```

Comment:
Mentioning here the check of overlapping CIDRs is too specific.
I think a more generic description would be better, as suggested in PR https://github.com/submariner-io/submariner-operator/pull/1173

Reply:
It's part of a Long description that is expected to be descriptive, isn't it? 
Also, in the other PR it includes the following description - "This command checks that the submariner components are properly deployed and running" . When overlapping CIDRs are present, Submariner Pods continue to run and only when users look into the logs of the Route-agent Pod, will know that there is some issue. 

Anyways, this is subjective thing. Lets hear from others and if the majority feel that overlapping CIDRs should be removed from description, I'll update the PR accordingly.

Reply:
Okay, my suggestion is - "This command checks that the submariner components are properly deployed, running and there are no overlapping CIDRs"

@tpantelis @dfarrell07 any suggestions?

Reply:
That sounds good to me @sridhargaddam :+1:

Reply:
Sounds good to me. Thanks @sridhargaddam